project.clj

```diff
@@ -32,15 +33,30 @@
   :profiles {
              ;; Clojure versions matrix
              :provided {:dependencies [[org.clojure/clojure "1.10.0"]]}
-             :1.8 {:dependencies [[org.clojure/clojure "1.8.0"]]}
-             :1.9 {:dependencies [[org.clojure/clojure "1.9.0"]]}
-             :1.10 {:dependencies [[org.clojure/clojure "1.10.0"]]}
+             :1.8 {:dependencies [[org.clojure/clojure "1.8.0"]
+                                  [org.clojure/clojurescript "1.10.520" :scope "provided"]]}
+             :1.9 {:dependencies [[org.clojure/clojure "1.9.0"]
+                                  [org.clojure/clojurescript "1.10.520" :scope "provided"]]}
+             :1.10 {:dependencies [[org.clojure/clojure "1.10.0"]
+                                   [org.clojure/clojurescript "1.10.520" :scope "provided"]]}
              :master {:repositories [["snapshots"
                                       "https://oss.sonatype.org/content/repositories/snapshots"]]
-                      :dependencies [[org.clojure/clojure "1.11.0-master-SNAPSHOT"]]}
+                      :dependencies [[org.clojure/clojure "1.11.0-master-SNAPSHOT"]
+                                     [org.clojure/clojurescript "1.10.520" :scope "provided"]]}
 
              :sysutils {:plugins [[lein-sysutils "0.2.0"]]}
 
+             ;; DEV tools
+             :dev {:dependencies [[pjstadig/humane-test-output "0.9.0"]
```

Comment:
What do you use those for?

Reply:
to improve the test output - this gives nice diffs - https://github.com/pjstadig/humane-test-output#clojure

Reply:
Can of course take it out if you don't like it...it is a `:dev` dep anyways...